spec/features/restrooms_spec.rb

```diff
@@ -59,7 +60,7 @@
       mock_location "Oakland"
       click_button 'Search by Current Location'
 
-      expect(page).not_to have_content 'Some Cafe'
+      expect(page).to have_content 'Some Cafe'
```

Comment:
I suppose this `expect()` may have been written "wrong" (with inverted boolean value), so it would pass originally, given the location aspects not being set up properly as the test scenario was originally written...

That said, given how strange the commit history of this test is, it would be reassuring and add to my confidence this is working correctly, if there was another restroom added for San Francisco, which we expect _not_ to show for a search in Oakland.

Something like this:

```rb
      create(:restroom, name: "Some San Francisco Bookshop")
```

...

```rb
      expect(page).not_to have_content 'Some San Francisco Bookshop'
```

Assuming that would work. Not strictly a blocker, and I can try to add this outside of the current Pull Request if that would be preferable. If I don't hear back I'll try and implement it myself when I get a moment to fire up Docker.